### ISSUE TITLE
Multi-layer Perceptron (MLP)

### ISSUE THREAD

**Issue by dianzrong** [uoft-tmi/signature-forgery-detection#7]
### Purpose
To try applying a different model architecture to detecting forged signatures. Note: MLPs can take a long time to train 
### Task
- [ ] Understand, experiment with, and implement the structure for the MLP
- [ ] Update the model to take in the processed images and return whether the signature is a forgery
- [ ] Train the model on the train dataset
- [ ] Fine-tune the model (test different numbers of hidden neurons, layers, iterations, etc.) on the validation dataset
- [ ] Report the model's accuracy on the test set

### Resources
- [Paper that uses MLP to detect forgeries](https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=0000416bcad4fc85a055ff95047a1148f55aaae7#page=273)
- [Intro to MLPs](https://www.pycodemates.com/2023/01/multi-layer-perceptron-a-complete-overview.html)
- [More info on MLPs](https://scikit-learn.org/stable/modules/neural_networks_supervised.html#:~:text=MLP%20requires%20tuning%20a%20number,is%20sensitive%20to%20feature%20scaling.)
- [Documentation for function by scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPClassifier.html)
